connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -93,13 +95,15 @@
     private static final Logger log = LoggerFactory.getLogger(DistributedHerder.class);
 
     private static final long RECONFIGURE_CONNECTOR_TASKS_BACKOFF_MS = 250;
+    private static final int BULK_THREAD_POOL_SIZE = 8;
```

Comment:
Pretty arbitrary choice. How far we would go in terms of flexibility (ranging from another config option to an non-fixed/unbounded thread pool)?